activemq-kahadb-store/src/test/java/org/apache/activemq/store/kahadb/disk/page/TransactionTest.java

```diff
@@ -0,0 +1,76 @@
+package org.apache.activemq.store.kahadb.disk.page;
```

Comment:
You are missing the apache license header

Reply:
I just  updated the PR! :D